spec/http-connection-tester.spec.ts

```diff
@@ -0,0 +1,84 @@
+
+describe("Given http connection tester", () => {
+    // tslint:disable-next-line:max-line-length
+    describe ("When I send a HTTP request with failOnHTTP enabled ", () => {
+        describe("and the website is not configured to redirect the user to HTTPS", () => {
+            const events  = {
+                failOnHTTP : true,

```

Comment:
`failOnHTTP` - is this property required?

There are no specs on the expected behaviour when the flag is disabled.

The expected of the website is the responsibility of the client?

Is a simple true/false response for this connection tester sufficient?

Reply:
Conversation - approved - simple solution for current use-case and keeps consistency with other connection testers